Documentation/docs/get-started/Overview.md

```diff
@@ -4,7 +4,7 @@ This pages explains the most important terms and elements of the AsTeRICS framew
 
 ## Solutions
 
-The AsTeRICS framework provides some selected [solutions](/solutions/) which can be used out of the box. The solutions are grouped by **use cases** and can be directly started from the solutions page. Some solutions depend on certain [input devices](#input-device) or other hardware and software. Just click onto the ```Read More``` button in a solution's tile to get a detailed step by step instruction of how to install and use it.
+The AsTeRICS framework provides some selected [solutions](/solutions/) which can be used out of the box. The solutions are grouped by **use cases** and can be directly started from the solutions page. Some solutions depend on certain [input devices](#input-device) or other hardware and software. Just click on the ```Read More``` button in a solution's tile to get a detailed step by step instruction of how to install and use it.
```

Comment:
click "on" should be correct, see https://dict.leo.org/forum/viewGeneraldiscussion.php?idforum=4&idThread=1182156&lp=ende&lang=de

Reply:
ok